distro_data.json

```diff
@@ -518,18 +518,16 @@
             "eol": "2017-05"
         },
         "versions": {
-            "kernel": "4.1",
-            "GCC": [
-                "4.8",
-                "5.2"
-            ],
-            "clang": "3.7",
+            "pixman": "0.32",
             "glib": "2.44",
             "glibc": "2.19",
+            "GCC": [ "4.8", "5.2" ],
+            "stdc++": "4.8",
```

Comment:
Ok, I know I said the .z didn't matter, but it actually does in exactly one case: libstdc++ 4.8 (I'm on my way to update the other distros that have a 4.8).

Reply:
BTW, this is a different version than what was listed before, is that actually true? (I got the current info from distrowatch, fwiw)

Reply:
And just to be extra clear, I'm interested in the version of libstdc++6, not libstdc++-devel.

Reply:
You are correct, that is wrong. I had a wrong assumption about which version goes into the default installation. Trying to find a non-annoying way to find out the correct one.